content/items.yaml

```diff
@@ -0,0 +1,23 @@
+  isActive: false
+
+- question: What are the differences between the methods?
+  answer: There are several differences between the constraint methods used in this analysis, including the kind of observational data they use and how the results are handled statistically. For instance, the UKCP and ClimWIP methods both use multiple observed variables (eg. temperature, precipitation) to constrain their projections, rather than just using the target variable being reported. Some methods constrain their variables at different spatial scales, from global to local, before deriving constrained local projections. Different methods also handle uncertainty in the projections differently. The following table includes the key characteristics of the different methods and is reprinted from Brunner et al. (2020), which contains more information on the differences between these methods.
+  isActive: false
+  figure: methods.png
+
+- question: What do the percentiles mean?
+  answer: A percentile is the level below which a given percentage of simulations fall when they are put in order according to a variable such as temperature. For instance, the 50th percentile in the atlas includes the lower 50% of the climate projections. The 10th percentile of precipitation projections may show significant drying, while the 90th percentile might show much wetter conditions.
+  isActive: false
+
+- question: Which method is best?
+  answer: |
+    The question of which constraint method is best depends on what the projection is being used for. Some methods have particular strengths over others.  For example, some methods provide relative weights of available simulations that could be used to select models to drive downstream climate impact analysis, and some methods can provide physically consistent projections for a number of variables.
+    Method performance can be different in different areas or with different variables. One method, for instance, could bring significant improvements over unconstrained projections in northern Europe but not in the Mediterranean region.   It should be noted, however, that all the constraint methods used in the Atlas confer some improvement over unconstrained projections for summer temperature, and strongly related variables, and most provide improvements for all regions.
+  isActive: false
+
+- question: Where can I find more information?
+  answer: Please view the references section below for more on the background research behind this Atlas. For more on the EUCP project, please head to our website. This Atlas has been produced by Work Package 2 of EUCP, which is described in more detail here.
+  isActive: false
```

Comment:
Could we include references questions, like

- Can I use these figures (yes, with attribution)
- I'd like to cite the Atlas in my paper, how can I do this?
- Is the Atlas data available as well?